### ISSUE TITLE
Q: Is vim-markdown-toc abandoned, dead?  (A: Yes)

### ISSUE THREAD

**Issue by WaterSibilantFalling** [mzlogin/vim-markdown-toc#76]
Last PR over 18 months ago.

Last closed issue over a year ago. 

No responses to any Issues. 

vim-markdown-toc is not working at all for me (installled, but vim will not acknowledge that it is there), but no point in raising an issue.

**Comment by mbpowers** [comment]
It is working for me, not to say that it is not abandoned/unmaintained.
I use nvim with vim plug, added `Plug 'mzlogin/vim-markdown-toc'` to `init.vim` and then did `:PlugInstall`.

**Comment by reckel-jm** [comment]
Same for me. The commands can't be found in nvim.

**Comment by mzlogin** [comment]
> Same for me. The commands can't be found in nvim.
> 
> **Update**: In my case, a change in the Plug-Configuration in my `~/.config/nvim` worked and solved the problem (I am using `nvim`):
> 
> ```
> call plug#begin('~/.config/nvim/plugged')
>     Plug 'mzlogin/vim-markdown-toc'
> call plug#end()
> ```
> 
> The important part here was to specify the file path in the begin function. After reloading the config, all plugins have to be installed again. However, afterwards the commands can be found and run well.

Yeah, what you do is how to install this plugin with vim-plug, documented in <https://github.com/junegunn/vim-plug#usage>.

**Comment by mzlogin** [comment]
@WaterSibilantFalling I'm sorry for my late reply. Could you provide more details? Like how you manage your vim plugins, what's in your vimrc, etc.

**Comment by dariober** [comment]
> vim-markdown-toc is not working at all for me (installled, but vim will not acknowledge that it is there)

Check your file has actually extension .md or .markdown extension (#86).
